db/migrate/20170704165711_add_user_bio_and_token.rb

```diff
@@ -0,0 +1,26 @@
+class AddUserBioAndToken < ActiveRecord::Migration
+  def up
+    add_column :rusers, :bio, :text, limit: 2147483647, default: ''
```

Comment:
Apparently default values are not universally supported??? whoa. I'll try setting it manually.